packages/grafana-ui/src/utils/logger.ts

```diff
@@ -19,7 +19,7 @@ export interface Logger {
 
 /** @internal */
 export const createLogger = (name: string): Logger => {
-  let LOGGIN_ENABLED = false;
+  let LOGGIN_ENABLED = true;
```

Comment:
Why did we need to change it to enabled by default?

Reply:
ugh sorry, my bad commit. 

this should be driven by config rather than fiddling code and remember to not commit it.